website/cue/reference/components/sinks/datadog_traces.cue

```diff
@@ -0,0 +1,79 @@
+package metadata
+
+components: sinks: datadog_traces: {
```

Comment:
Should we add a warning to this component about the lack of APM stats support? I can see users being confused when they try to use `datadog_agent` -> `datadog_traces` and are missing data in Datadog.